lib/perl/Genome/Site/TGI/Extension/ConfigureQueuedInstrumentData.pm

```diff
@@ -0,0 +1,29 @@
+package Genome::Config::Command::ConfigureQueuedInstrumentData;
+
+use strict;
+use warnings;
+
+use Genome;
+use Genome::Config::Command::ConfigureQueuedInstrumentData; #load real module first
+
+use Sub::Install;
+
+Sub::Install::reinstall_sub ({
+    as => '_import_instrument_data',
+    code => \&_import_lims_data,
+
+});
+
+sub _import_lims_data {
```

Comment:
Is there a particular reason to use `Sub::Install` here rather than normal inheritance with an overridden method (or similar?)

Reply:
`Sub::Install` is a nice way to avoid the method-redefinition warning.  The site extensions are kind of class monkey-patching for things peculiar to a particular location.  In this case, nobody else in the world has this LIMS system!  There's no subclassing happening, so I'm not sure what it would look like to try to use inheritance here.

(It's an open question to me whether it's worthwhile to maintain the notion that there is both a core system-independent GMS and also site-specific extensions (especially since it's partially broken already with some of the `Genome::Sys` extensions in use).  As long as it's not hard to keep up, I'm not motivated to intentionally break it.)

Reply:
Cool, makes sense to me - I was imagining `Site::TGI::Command::CQID` as possibly just being a subclass of main `CQID` that implements this method, but this makes sense given the context. Looks good to me!